packages/core/package.json

```diff
@@ -35,9 +35,9 @@
     "@dotkomonline/types": "workspace:*",
     "@testcontainers/postgresql": "^10.7.1",
     "@types/node": "^18.18.13",
-    "@vitest/ui": "^0.34.6",
+    "@vitest/ui": "^1.3.1",
```

Comment:
If we use vitest anywhere else, it should also be updated at the same time (to avoid having multiple vitests in the codebase)